# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bzip2/makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Sat Jan 26 00:34:53 2002
+++ ./Makefile	Wed Apr  3 01:35:16 2002
@@ -12,8 +12,8 @@
 CFLAGS=-Wall -Winline -O2 -fomit-frame-pointer -fno-strength-reduce $(BIGFILES)
 
 # Where you want it installed when you do 'make install'
-PREFIX=/usr
-
+PREFIX=/
+USRPREFIX=/usr
 
 OBJS= blocksort.o  \
       huffman.o    \
@@ -23,7 +23,7 @@
       decompress.o \
       bzlib.o
 
-all: libbz2.a bzip2 bzip2recover test
+all: libbz2.a bzip2 bzip2recover
 
 bzip2: libbz2.a bzip2.o
 	$(CC) $(CFLAGS) $(LDFLAGS) -o bzip2 bzip2.o -L. -lbz2
@@ -34,11 +34,7 @@
 libbz2.a: $(OBJS)
 	rm -f libbz2.a
 	$(AR) cq libbz2.a $(OBJS)
-	@if ( test -f $(RANLIB) -o -f /usr/bin/ranlib -o \
-		-f /bin/ranlib -o -f /usr/ccs/bin/ranlib ) ; then \
-		echo $(RANLIB) libbz2.a ; \
-		$(RANLIB) libbz2.a ; \
-	fi
+	$(RANLIB) libbz2.a
 
 check: test
 test: bzip2
@@ -59,42 +55,42 @@
 
 install: bzip2 bzip2recover
 	if ( test ! -d $(PREFIX)/bin ) ; then mkdir -p $(PREFIX)/bin ; fi
-	if ( test ! -d $(PREFIX)/lib ) ; then mkdir -p $(PREFIX)/lib ; fi
-	if ( test ! -d $(PREFIX)/man ) ; then mkdir -p $(PREFIX)/man ; fi
-	if ( test ! -d $(PREFIX)/man/man1 ) ; then mkdir -p $(PREFIX)/man/man1 ; fi
-	if ( test ! -d $(PREFIX)/include ) ; then mkdir -p $(PREFIX)/include ; fi
+	if ( test ! -d $(USRPREFIX)/lib ) ; then mkdir -p $(USRPREFIX)/lib ; fi
+	if ( test ! -d $(USRPREFIX)/man ) ; then mkdir -p $(USRPREFIX)/man ; fi
+	if ( test ! -d $(USRPREFIX)/man/man1 ) ; then mkdir -p $(USRPREFIX)/man/man1 ; fi
+	if ( test ! -d $(USRPREFIX)/include ) ; then mkdir -p $(USRPREFIX)/include ; fi
 	cp -f bzip2 $(PREFIX)/bin/bzip2
 	cp -f bzip2 $(PREFIX)/bin/bunzip2
 	cp -f bzip2 $(PREFIX)/bin/bzcat
-	cp -f bzip2recover $(PREFIX)/bin/bzip2recover
+	cp -f bzip2recover $(USRPREFIX)/bin/bzip2recover
 	chmod a+x $(PREFIX)/bin/bzip2
 	chmod a+x $(PREFIX)/bin/bunzip2
 	chmod a+x $(PREFIX)/bin/bzcat
-	chmod a+x $(PREFIX)/bin/bzip2recover
-	cp -f bzip2.1 $(PREFIX)/man/man1
-	chmod a+r $(PREFIX)/man/man1/bzip2.1
-	cp -f bzlib.h $(PREFIX)/include
-	chmod a+r $(PREFIX)/include/bzlib.h
-	cp -f libbz2.a $(PREFIX)/lib
-	chmod a+r $(PREFIX)/lib/libbz2.a
-	cp -f bzgrep $(PREFIX)/bin/bzgrep
-	ln $(PREFIX)/bin/bzgrep $(PREFIX)/bin/bzegrep
-	ln $(PREFIX)/bin/bzgrep $(PREFIX)/bin/bzfgrep
-	chmod a+x $(PREFIX)/bin/bzgrep
-	cp -f bzmore $(PREFIX)/bin/bzmore
-	ln $(PREFIX)/bin/bzmore $(PREFIX)/bin/bzless
-	chmod a+x $(PREFIX)/bin/bzmore
-	cp -f bzdiff $(PREFIX)/bin/bzdiff
-	ln $(PREFIX)/bin/bzdiff $(PREFIX)/bin/bzcmp
-	chmod a+x $(PREFIX)/bin/bzdiff
-	cp -f bzgrep.1 bzmore.1 bzdiff.1 $(PREFIX)/man/man1
-	chmod a+r $(PREFIX)/man/man1/bzgrep.1
-	chmod a+r $(PREFIX)/man/man1/bzmore.1
-	chmod a+r $(PREFIX)/man/man1/bzdiff.1
-	echo ".so man1/bzgrep.1" > $(PREFIX)/man/man1/bzegrep.1
-	echo ".so man1/bzgrep.1" > $(PREFIX)/man/man1/bzfgrep.1
-	echo ".so man1/bzmore.1" > $(PREFIX)/man/man1/bzless.1
-	echo ".so man1/bzdiff.1" > $(PREFIX)/man/man1/bzcmp.1
+	chmod a+x $(USRPREFIX)/bin/bzip2recover
+	cp -f bzip2.1 $(USRPREFIX)/man/man1
+	chmod a+r $(USRPREFIX)/man/man1/bzip2.1
+	cp -f bzlib.h $(USRPREFIX)/include
+	chmod a+r $(USRPREFIX)/include/bzlib.h
+	cp -f libbz2.a $(USRPREFIX)/lib
+	chmod a+r $(USRPREFIX)/lib/libbz2.a
+	cp -f bzgrep $(USRPREFIX)/bin/bzgrep
+	ln -f $(USRPREFIX)/bin/bzgrep $(USRPREFIX)/bin/bzegrep
+	ln -f $(USRPREFIX)/bin/bzgrep $(USRPREFIX)/bin/bzfgrep
+	chmod a+x $(USRPREFIX)/bin/bzgrep
+	cp -f bzmore $(USRPREFIX)/bin/bzmore
+	ln -f $(USRPREFIX)/bin/bzmore $(USRPREFIX)/bin/bzless
+	chmod a+x $(USRPREFIX)/bin/bzmore
+	cp -f bzdiff $(USRPREFIX)/bin/bzdiff
+	ln -f $(USRPREFIX)/bin/bzdiff $(USRPREFIX)/bin/bzcmp
+	chmod a+x $(USRPREFIX)/bin/bzdiff
+	cp -f bzgrep.1 bzmore.1 bzdiff.1 $(USRPREFIX)/man/man1
+	chmod a+r $(USRPREFIX)/man/man1/bzgrep.1
+	chmod a+r $(USRPREFIX)/man/man1/bzmore.1
+	chmod a+r $(USRPREFIX)/man/man1/bzdiff.1
+	echo ".so man1/bzgrep.1" > $(USRPREFIX)/man/man1/bzegrep.1
+	echo ".so man1/bzgrep.1" > $(USRPREFIX)/man/man1/bzfgrep.1
+	echo ".so man1/bzmore.1" > $(USRPREFIX)/man/man1/bzless.1
+	echo ".so man1/bzdiff.1" > $(USRPREFIX)/man/man1/bzcmp.1
 
 distclean: clean
 clean: 
